README.md

```diff
@@ -102,27 +102,29 @@ pub fn main() -> BlockResult {
+| Block             | Description                                                                                              |
+|:------------------|:---------------------------------------------------------------------------------------------------------|
+| [`Buffer`]        | Stores all messages it receives.                                                                         |
+| [`Const`]         | Sends a constant value.                                                                                  |
+| [`Count`]         | Counts the number of messages it receives, while optionally passing them through.                        |
+| [`Decode`]        | Decodes messages from a byte stream.                                                                     |
+| [`DecodeJSON`]    | Decodes JSON messages from a byte stream.                                                                |
+| [`Delay`]         | Passes messages through while delaying them by a fixed or random duration.                               |
+| [`Drop`]          | Discards all messages it receives.                                                                       |
+| [`Encode`]        | Encodes messages to a byte stream.                                                                       |
+| [`EncodeHex`]     | Encodes a byte stream into hexadecimal form.                                                             |
+| [`EncodeJSON`]    | Encodes messages into JSON format.                                                                       |
+| [`Hash`]          | Computes the cryptographic hash of a byte stream.                                                        |
+| [`Random`]        | Generates and sends a random value.                                                                      |
+| [`ReadDir`]       | Reads file names from a file system directory.                                                           |
+| [`ReadEnv`]       | Reads the value of an environment variable.                                                              |
+| [`ReadFile`]      | Reads bytes from the contents of a file.                                                                 |
+| [`ReadStdin`]     | Reads bytes from standard input (aka stdin).                                                             |
+| [`WriteFile`]     | Writes or appends bytes to the contents of a file.                                                       |
+| [`WriteStderr`]   | Writes bytes to standard error (aka stderr).                                                             |
+| [`WriteStdout`]   | Writes bytes to standard output (aka stdout).                                                            |
+| [`ConcatStrings`] | Concatenates the received string messages, with an optional joiner string inserted between each message. |
+| [`SplitString`]   | Splits the received input message, with an optional delimiter string parameter.                          |
```

Comment:
These should be in alphabetical order, and consistent with each other: hence "joiner" should be "delimiter"

Reply:
done